lineman/app/components/thread_preview/thread_preview.haml

```diff
@@ -11,9 +11,8 @@
       %i.fa.fa-times>
       %span{translate: 'volume_levels.mute'}
 
-  %a.thread-preview__link{href: '/d/{{ thread.key }}'}
+  %a.thread-preview__link{lmo-href: '', lmo-href-model: 'thread'}
```

Comment:
that's really cool. I agree that overloading lmo-href with string or model is not as nice as lm-href-model.. I wonder if you're overloading it again with this last (cool) feature. May I suggest you consider:

```
%a{lmo-href-model: "[group, 'memberships']"}
```

like how rails does it, for this last case?